packages/EasyErrorHandler/src/Bridge/Symfony/Builder/ApiPlatformErrorResponseBuilder.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Builder;
+
+use ApiPlatform\Core\Bridge\Symfony\Validator\Exception\ValidationException;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
+use EonX\EasyErrorHandler\Builders\AbstractErrorResponseBuilder;
+use EonX\EasyErrorHandler\Interfaces\TranslatorInterface;
+use Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException;
+use Symfony\Component\Serializer\Exception\UnexpectedValueException;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Throwable;
+
+final class ApiPlatformErrorResponseBuilder extends AbstractErrorResponseBuilder
```

Comment:
I believe we shouldn't collapse two builders into one. I'd suggest the following classnames and its responsibilities:

 - `ApiPlatformValidationExceptionResponseBuilder` is responsible for building correct response for `ValidationException`.
 - `ApiPlatformValidationErrorResponseBuilder` is responsible for building correct response for any error that is treated as validation exception.

Extra points for this idea:
 - `ApiPlatformValidationErrorResponseBuilder::supports` makes no sense for `ValidationException` because it's already ignored by `bugsnagIgnoredExceptions` configuration
 - `ApiPlatformValidationExceptionResponseBuilder` doesn't need any `MESSAGE_PATTERN_...` consts for its logic

Splitting of these responsibilities would make code cleaner I believe.

Reply:
Done. Please check